### DISCUSSION TITLE
IdentityReplicationPolicy

### DISCUSSION THREAD

**Discussion by AEncinas19** [strimzi/strimzi-kafka-operator#8004] [Q&A] [answered]
Why with the IdentityReplicationPolicy the number of replicas should be higher than the number of min.insync.replicas?
If I put 1 replicas and 1 min.insync.replicas when I create a topic, when I write something in the topic, it appears an error of NOT_ENOUGH_REPLICAS.

**Comment by scholzj** [comment, marked as answer]
I don't think the `IdentityReplicationPolicy` has anything to do with it. MM2 in general does not _mirror_ your replication settings. That is fully in control of the target cluster's default. If you want the topics to have some special replication settings, you should manage them manually on the target cluster.
